aten/src/ATen/native/native_functions.yaml

```diff
@@ -3267,12 +3267,20 @@
   dispatch:
     CompositeExplicitAutograd: linspace
 
+- func: linspace.Tensor(Tensor start, Tensor end, int steps, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
+  category_override: factory
+  dispatch:
+    CompositeExplicitAutograd: linspace
```

Comment:
We're going to need to add a meta registration for linspace. We should add one in https://github.com/pytorch/pytorch/blob/main/torch/_meta_registrations.py.

Concretely, the problem we're facing right now is that we've marked linspace.Tensor as CompositeExplicitAutograd. This automatically gives linspace.Tensor a meta registration that runs `linspace`.

Instead of doing that, we just want the meta registration to return torch.empty(steps, dtype, layout, device, pin_memory) (or steps+1? I'm not sure).